src/lib.rs

```diff
@@ -607,17 +608,46 @@ impl StructuredExtendedInformation {
         13 => deprecates_fpu_cs_ds,
         // 14 - reserved
         15 => pqe,
-        // 16-17 - reserved
+        16 => avx512f,
+        17 => avx512dq,
         18 => rdseed,
         19 => adx,
         20 => smap,
-        // 21-24 - reserved
-        25 => intel_processor_trace
-        // 26-31 - reserved
+        21 => avx512ifma,
+        // 22-24 - TODO
```

Comment:
Should add:

> Bit 22: Reserved.
> Bit 23: CLFLUSHOPT.
> Bit 24: CLWB.

Reply:
I left everything non-AVX512 related out of this PR on purpose (basically everywhere where I wrote TODO). Should I complete those?